File: docs/src/main/java/zkLoginJvm.java
zkLogin(req, new ZKLoginFeedback() {
    @Override
    public void onToken(Option<String> option) {
        System.out.println("onToken: " + option);
    }

    @Override
    public void onProof(Option<String> option) {
        System.out.println("onProof: " + option);
    }

    @Override
    public void onSalt(Option<String> option) {
        System.out.println("onSalt: " + option);
    }
});

File: docs/src/main/kotlin/zkLoginJvm.kt
zkLogin(
    req,
    object : ZKLoginFeedback {
      override fun onToken(option: Option<String>) {
        println("onToken: $option")
      }

      override fun onProof(option: Option<String>) {
        println("onProof: $option")
      }

      override fun onSalt(option: Option<String>) {
        println("onSalt: $option")
      }
    },
  )
